mbed_build/_internal/mbed_tools/export.py

```diff
@@ -37,9 +39,11 @@ def export(output_directory: str, toolchain: str, mbed_target: str) -> None:
         output_directory: where the top-level CMakeLists.txt should be exported to
         toolchain: the toolchain you are using to build your app (eg. GCC, ARM5 etc.)
         mbed_target: the build target you are wanting to run your app (eg. K64F)
+        targets_json_path: the path to targets.json file in Mbed OS library
 
     Raises:
         InvalidExportOutputDirectory: it's not possible to export to the output directory provided
     """
-    cmake_file_contents = generate_cmakelists_file(mbed_target, toolchain)
+    target_build_attributes = get_build_attributes_by_board_type(mbed_target, targets_json_path)
```

Comment:
This should be `project_directory` and not `targets_json_path`.

Reply:
e1433ca